app/models/manageiq/providers/ibm_cloud/inventory/parser/power_virtual_servers.rb

```diff
@@ -211,7 +211,8 @@ def systemtypes
     collector.system_pool.each do |v|
       persister.flavors.build(
         :type    => "ManageIQ::Providers::IbmCloud::PowerVirtualServers::CloudManager::SystemType",
-        :ems_ref => v['type']
+        :ems_ref => v['type'],
+        :name => v['type']
```

Comment:
Can you line up the `=>` ?

Reply:
D'oh too slow https://github.com/ManageIQ/manageiq-providers-ibm_cloud/pull/44#issuecomment-697956776

Reply:
> but you can still beat him to it

Winner: @miq-bot  :smile: